include/static-pages.php

```diff
@@ -56,6 +56,8 @@ public function __construct( &$polylang ) {
 
 		$this->init();
 
+		add_filter( 'pll_static_pages', array( $this, 'set_static_pages' ), 10, 2 );
```

Comment:
Maybe we can add it in `$this->init()` and add the hook only if `'page' == get_option( 'show_on_front' )`?

Reply:
Done in 1d5dcf3f3fb67570577444c331a3416724e0e433